content/compose/environment-variables/variable-interpolation.md

```diff
@@ -211,3 +212,42 @@ If an environment variable is not set, Compose substitutes with an empty string.
 > **Note**
 >
 > `postgres:` is not a valid image reference. Docker expects either a reference without a tag, like `postgres` which defaults to the latest image, or with a tag such as `postgres:15`.
+
+
+#### local `.env` file _vs_ <project directory> `.env` file
+
+An `.env` file can also be used to declare [`COMPOSE_*` variables](envvars.md) used to control Compose behavior and files to be loaded. When executed without an
+explicit `--env-file` flag, Compose will search for a `.env` file in working directory (PWD) and load values both for self-configuration and interpolation. If
```

Comment:
if you mean the command -> `pwd` I would change it to this and maybe add a reference https://man7.org/linux/man-pages/man1/pwd.1.html